db/schema.rb

```diff
@@ -698,6 +1603,59 @@
+  add_foreign_key "projects_source_research_questions", "projects_researchquestion", column: "researchquestion_id", name: "projects_source_rese_researchquestion_id_aa32edb1_fk_projects_", deferrable: :deferred
+  add_foreign_key "projects_source_research_questions", "projects_source", column: "source_id", name: "projects_source_rese_source_id_451782de_fk_projects_", deferrable: :deferred
+  add_foreign_key "projects_source_topics_covered", "projects_source", column: "source_id", name: "projects_source_topi_source_id_d120fe6e_fk_projects_", deferrable: :deferred
+  add_foreign_key "projects_source_topics_covered", "projects_topics", column: "topics_id", name: "projects_source_topi_topics_id_83f74664_fk_projects_", deferrable: :deferred
+  add_foreign_key "resources_category", "resources_category", column: "parent_category_id", name: "resources_category_parent_category_id_6fffa3b5_fk_resources", deferrable: :deferred
+  add_foreign_key "resources_resource", "datasources_parcel", column: "parcel_id", name: "resources_resource_parcel_id_d8e5f681_fk_datasources_parcel_id", deferrable: :deferred
+  add_foreign_key "resources_resource", "datasources_refresh", column: "refresh_id", name: "resources_resource_refresh_id_a263997c_fk_datasourc", deferrable: :deferred
+  add_foreign_key "resources_resource", "resources_resourcelisthtml", column: "list_html_id", name: "resources_resource_list_html_id_93af9cdb_fk_resources", deferrable: :deferred
+  add_foreign_key "resources_resource_categories", "resources_category", column: "category_id", name: "resources_resource_c_category_id_3944a462_fk_resources", deferrable: :deferred
+  add_foreign_key "resources_resource_categories", "resources_resource", column: "resource_id", name: "resources_resource_c_resource_id_f2f988e9_fk_resources", deferrable: :deferred
+  add_foreign_key "resources_resourcelisthtml", "datasources_refresh", column: "refresh_id", name: "resources_resourceli_refresh_id_f68a7098_fk_datasourc", deferrable: :deferred
+  add_foreign_key "socialaccount_socialaccount", "auth_user", column: "user_id", name: "socialaccount_socialaccount_user_id_8146e70c_fk_auth_user_id", deferrable: :deferred
+  add_foreign_key "socialaccount_socialapp_sites", "django_site", column: "site_id", name: "socialaccount_social_site_id_2579dee5_fk_django_si", deferrable: :deferred
+  add_foreign_key "socialaccount_socialapp_sites", "socialaccount_socialapp", column: "socialapp_id", name: "socialaccount_social_socialapp_id_97fb6e7d_fk_socialacc", deferrable: :deferred
+  add_foreign_key "socialaccount_socialtoken", "socialaccount_socialaccount", column: "account_id", name: "socialaccount_social_account_id_951f210e_fk_socialacc", deferrable: :deferred
+  add_foreign_key "socialaccount_socialtoken", "socialaccount_socialapp", column: "app_id", name: "socialaccount_social_app_id_636a42d7_fk_socialacc", deferrable: :deferred
```

Comment:
This is a big schema change that is not related to this PR. Let's drop this so that our schema file stays clean 🧹